### DISCUSSION TITLE
How to do a Custom entry page instead of index.html

### DISCUSSION THREAD

**Discussion by Sid00721** [vercel/community#7117] [Help] [answered]
### Summary

I have a project with the files
data.json
display.php
home.html
profile.html
sbhsdemo.php

i want sbhsdemo.php to be the first page open in the deployment how do i do this
![image](https://github.com/vercel/community/assets/60917616/be506fe1-615b-48ff-ad9d-16936605f1ae)

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/60917616/02972ba5-4dcd-485e-b9f8-55a4b3d99dca)

**Comment by amyegan** [comment, marked as answer]
You can use [rewrites](https://vercel.com/docs/edge-network/rewrites#rewrites-on-vercel) or [redirects](https://vercel.com/docs/edge-network/redirects#redirects) show users a different page when when they come to the root domain
